src/main/java/markovchain/wanglandau/energy/WangLandauEnergy.kt

```diff
@@ -9,11 +9,6 @@ import markovchain.MarkovMove
 </E></MarkovState> */
 abstract class WangLandauEnergy<MarkovState, MM : MarkovMove<MarkovState>, E : WangLandauEnergy<MarkovState, MM, E>> {
 
-    /**
-     * @return A new WangLandauEnergy object holding the same value as this one.
-     */
-    abstract fun copy(): E
-
```

Comment:
The copy function was only used by `CompositeEnergy` and is deprecated by the usage of immutable types.